tests/StackExchange.Redis.Tests/Lists.cs

```diff
@@ -313,5 +313,23 @@ public async Task ListRightPushAsyncMultipleToExisitingKey()
                 Assert.Equal("testvalue3", rangeResult[2]);
             }
         }
+
+        [Fact]
+        public async Task ListMove()
+        {
+            using (var conn = Create())
+            {
+                var db = conn.GetDatabase();
+                RedisKey src = "testlist";
+                RedisKey dest = "destinationlist";
+                db.KeyDelete(src, CommandFlags.FireAndForget);
+
+                var pushResult = await db.ListRightPushAsync(src, new RedisValue[] { "testvalue1" });
+                Assert.Equal(1, pushResult);
+
+                var rangeResult = db.ListMove(src, dest, ListSide.Left, ListSide.Right);
+                Assert.Equal("testvalue1" , rangeResult);
+            }
```

Comment:
We should test both directions as well as what happens when the source/destination lists are null.